### PR TITLE
Fix LocalDateTime property test in mongodb std lib 

### PR DESCRIPTION
**Pull request by rintcius**
#qz-3778 Done

### CODE REVIEW

File: mongoIt/src/test/scala/quasar/physical/mongodb/MongoDbStdLibSpec.scala [hunk 1]
```diff
@@ -84,7 +85,8 @@ abstract class MongoDbStdLibSpec extends StdLibSpec {
         Data.Str(time.toString)
       case Data.LocalDateTime(time) =>
         Data.OffsetDateTime(JOffsetDateTime.of(
-          time.withNano(scala.math.round(time.getNano.toDouble / 1000000).toInt * 1000000), // round to millis
+          time.truncatedTo(ChronoUnit.SECONDS)
+            .plusNanos(scala.math.round(time.getNano.toDouble / 1000000).toLong * 1000000), // round to millis
```
Comment (alissapajer) [inline mongoIt/src/test/scala/quasar/physical/mongodb/MongoDbStdLibSpec.scala]: What's the difference between `withNano` versus `truncatedTo` and `plusNanos`? I thought that `withNano` would be equivalent to those two. What difference did you find?

Comment (rintcius) [reply mongoIt/src/test/scala/quasar/physical/mongodb/MongoDbStdLibSpec.scala]: just a difference in a corner case: `withNano(1000000000)` fails with `Invalid value for NanoOfSecond (valid values 0 - 999999999): 1000000000` whereas `truncatedTo` & `plusNanos` allow it

Comment (alissapajer) [reply mongoIt/src/test/scala/quasar/physical/mongodb/MongoDbStdLibSpec.scala]: :+1: could you add a comment here that says that? Otherwise someone might revert it in the future.
